File: problems/a/akcija/akcija.cpp
/*
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
*/

#include <bits/stdc++.h>

using namespace std;

int main() {
    int n, total;

    cin >> n;

    vector<int> books;

    for (int i=0; i<n; i++) {
        int book;
        cin >> book;
        books.push_back(book);
    }

    sort(books.begin(), books.end(), greater<int>());

    total = 0;
    for (int i=0; i<n; i++) {
        if (i % 3 != 2) {
            total += books[i];
        }
    }

    cout << total << endl;
}

File: problems/a/akcija/akcija.py
#! /usr/bin/python3
"""
Author: Rudi César Comiotto Modena
Email: rudi.modena@gmail.com
"""

def main():
    n = int(input())

    books = list()
    for _ in range(n):
        books.append(int(input()))
    
    books.sort(reverse=True)
    print(sum([book for i, book in enumerate(books) if i % 3 != 2]))


if __name__ == '__main__':
    main()